.github/workflows/sentry-release.yml

```
@@ -0,0 +1,63 @@
+    branches:
+      - main
+      - develop
+
+jobs:
+  sentry-release:
+    name: Sentry Release
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)